backupstoragelocation.md

```diff
@@ -40,4 +40,11 @@ spec:
     #
     # Optional.
     subscriptionId: my-subscription
+
+    # The block size, in bytes, to use when uploading objects to Azure blob storage.
+    # See https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs#about-block-blobs
+    # for more information on block blobs.
+    # 
+    # Optional (defaults to 104857600, i.e. 100MB).
+    blockSizeInBytes: "10485760"
```

Comment:
Should the value here be "104857600"? I think 10485760 corresponds to 10MB.

Reply:
Yeah, I put a non-default value in here since the field is not required if you just want the default. I could put something totally different (e.g. 1024) so it's not confusing

Reply:
Ahhhhhhh. Could it be left blank?

Reply:
eh, I'll just put the default value in. I'd like to have a sample value.

Reply:
Yes, that's good too.

Reply:
Updated.